NativeApp/src/screens/Booking.js

```diff
@@ -0,0 +1,23 @@
+import React, { Component } from 'react';
+import { PropTypes as PT } from 'prop-types';
+import Booking from '../components/Booking';
+
+export default class BookingScreen extends Component {
+  static navigationOptions = {
+    title: 'Request Holidays',
+  }
+
+  static propTypes = {
+    navigation: PT.shape({
+      navigate: PT.func,
+    }).isRequired,
+  }
+
+  render() {
+    const { navigation } = this.props;
+
+    return (
```

Comment:
I might be wrong here - but can this be written as this? `return <Booking navigation={navigation} />`

It would just be a bit cleaner

Reply:
Your totally right. Thought the linter would complain but didn't. Gonna convert the other screens as well.